stdlib/testing/testing.flux

```diff
@@ -342,3 +343,14 @@ benchmark = (case) => {
 
     return tc.input |> tc.fn()
 }
+
+// assertEquals tests whether two values are equal
+//
+// ## Parameters
+// - `got` is the value to test.
+// - `want` is the expected data to test against.
+//
+assertEqualValues = (got, want) => {
```

Comment:
I really like this addition, and we can update it once the testing framework is updated to not expect tables as output.